### PR TITLE
added desgin and ajax functionality for whitelist scritps

### PR DESCRIPTION
**Pull request by Omendra123Rajput**

### CODE REVIEW

File: admin/css/gdpr-cookie-consent-admin.css [hunk 1]
```diff
@@ -2643,6 +2643,364 @@ input.readonly, input[readonly], textarea.readonly, textarea[readonly] {
...
+
+.wpl-panel {
+	background: #fff;
+	padding: 20px 30px;
+	padding-top: 5px;
+  }
+  .wpl-slide-panel {
+	margin: 15px 0;
+	color: #383d41;
+	background-color: #D8DBE0;;
```
Comment (Nayan-Gaikwad) [inline admin/css/gdpr-cookie-consent-admin.css:2664]: @Omendra123Rajput Remove extra semicolon

File: src/vue-settings.js [hunk 2]
```diff
@@ -2240,6 +2240,110 @@ var gen = new Vue({
...
+				var type = 'whitelist_script';
+				btn.html('<div class="wpl-loader"><div class="rect1"></div><div class="rect2"></div><div class="rect3"></div><div class="rect4"></div><div class="rect5"></div></div>');
+
+				jQuery.ajax({
+					type: 'POST',
+					url: settings_obj.ajaxurl,
+					data: ({
+								action: 'wpl_script_add',
+								type: type,
+								// nonce: complianz_admin.nonce,
```
Comment (Nayan-Gaikwad) [inline src/vue-settings.js:2335]: @Omendra123Rajput Why nonce is commented

File: src/vue-settings.js [hunk 3]
```diff
@@ -2240,6 +2240,110 @@ var gen = new Vue({
...
+							}),
+							success: function (response) {
+										if (response.success) {
+											btn.before(response.html);
+											btn.html(btn_html);
+										}
+									}
+				}).done(function (data) {
+						//
```
Comment (Nayan-Gaikwad) [inline src/vue-settings.js:2344]: @Omendra123Rajput empty event

**Review by Nayan-Gaikwad — CHANGES_REQUESTED** [review]

File: src/vue-settings.js (hunk 3 above)
Comment (Omendra123Rajput) [reply src/vue-settings.js:2344]: left it empty for future if we want to add anything later

File: admin/css/gdpr-cookie-consent-admin.css (hunk 1 above)
Comment (Omendra123Rajput) [reply admin/css/gdpr-cookie-consent-admin.css:2664]: done

File: src/vue-settings.js (hunk 2 above)
Comment (Omendra123Rajput) [reply src/vue-settings.js:2335]: this was from competitor code

File: src/vue-settings.js (hunk 3 above)
Comment (Nayan-Gaikwad) [reply src/vue-settings.js:2343]: Okay

**Review by Nayan-Gaikwad — APPROVED** [review]

**Pull request merged**